runtime/shiden/src/lib.rs

```diff
@@ -756,40 +756,8 @@ pub type Executive = frame_executive::Executive<
     frame_system::ChainContext<Runtime>,
     Runtime,
     AllPalletsWithSystem,
-    (InitRewardConfigSettings,),
 >;
 
-pub struct InitRewardConfigSettings;
```

Comment:
If you change the runtime in any way, you have to increase the spec version.

My suggestion would be not to change this file at all in this PR since then we have an _empty_ spec_version bump - we're not going to do this upgrade.

Reply:
:+1: 

Now I see I forgot to say one more thing - you should rebase :laughing: